chapter_appendix_math/geometry-linear-algebric-ops_vn.md

```diff
@@ -148,33 +148,36 @@ we see that the vector $\mathbf{u}-\mathbf{v}$ is the direction
 that takes us from the point $\mathbf{u}$ to the point $\mathbf{v}$.
 -->
 
-*dịch đoạn phía trên*
+Hiệu của hai vector có một cách diễn giải tương tự.
+Bằng cách biểu diễn $\mathbf{u} = \mathbf{v} + (\mathbf{u}-\mathbf{v})$,
+ta thấy rằng vector $\mathbf{u}-\mathbf{v}$ là hướng mang điểm $\mathbf{u}$ tới
+điểm $\mathbf{v}$.
 
 
 <!--
 ## Dot Products and Angles
 -->
 
-## *dịch tiêu đề phía trên*
+## Tích vô hướng và Góc
+
+<!--
 As we saw in :numref:`sec_linear-algebra`,
 if we take two column vectors say $\mathbf{u}$ and $\mathbf{v}$,
 we can form their dot product by computing:
 -->
 
-*dịch đoạn phía trên*
+Như đã thấy trong :numref:`sec_linear-algebra`, tích vô hướng của hai vector cột
+$\mathbf{u}$ và $\mathbf{v}$ có thể được tính bởi:
 
 $$\mathbf{u}^\top\mathbf{v} = \sum_i u_i\cdot v_i.$$
 :eqlabel:`eq_dot_def`
--->
-
-*dịch đoạn phía trên*
 
 <!--
 Because :eqref:`eq_dot_def` is symmetric, we will mirror the notation
 of classical multiplication and write
 -->
 
-*dịch đoạn phía trên*
+Vì biểu thức :eqref:`eq_dot_def` đối xứng, chúng ta có thể viết:
```

Comment:
"biểu thức" --> "phép toán" 

Trong toán học, các phép cọng, trừ, tích vô hướng ... gọi là phép toán. Cách gọi này được phổ biến

Reply:
Cảm ơn @thoangtrvn vì rất review rất nhiều từ hôm qua tới bây giờ.

Mình có một gợi ý nhỏ là khi review bạn có thể dùng chức năng dưới đây, sẽ tiện hơn cho cả tác giả và người review:
![img](https://user-images.githubusercontent.com/19977/58752991-f39d0880-846c-11e9-8c03-c7aded86ee9b.png)

Reply:
Mình đã tham gia nhóm qua account Tuan (HTMT).